pkg/volume/gce_pd/gce_util_test.go

```diff
@@ -21,13 +21,27 @@ import (
 	"testing"
 
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util/exec"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util/mount"
 )
 
+type ErrorMounter struct {
+	mount.FakeMounter
+	err error
+}
+
+func (mounter ErrorMounter) Mount(source string, target string, fstype string, options []string) error {
+	if mounter.err != nil {
+		return mounter.err
+	}
+	return mounter.FakeMounter.Mount(source, target, fstype, options)
+}
+
 func TestSafeFormatAndMount(t *testing.T) {
```

Comment:
Hmm.. I've thought about that. The problem is that it is difficult to do without being able to mock 'os'. It would be worthwhile to refactor persistent volumes so that the os object is injected and can be mocked for testing but that would have to be its own patch.